package.json

```diff
@@ -249,7 +249,7 @@
     "brfs": "^2.0.2",
     "browserify": "^16.5.1",
     "chalk": "^3.0.0",
-    "chromedriver": "^79.0.0",
+    "chromedriver": "^87.0.1",
```

Comment:
[Q] What's the basis for upgrading to this particular version? It looks like latest is `92.0.2`, matching current chrome version?

Reply:
The reasoning was included in the PR description - this was the minimum version that included a fix for the bug that was encountered. I agree we should probably update this further though. Maybe in a future PR?

Reply:
Yeah, once this is through I will submit another PR to bring it up to the latest version so that we can run Chrome tests locally.